docs/docs/100-reference/01-command-line/acorn_run.md

```diff
@@ -12,99 +12,48 @@ acorn run [flags] IMAGE|DIRECTORY [acorn args]
-      --compute-class strings     Set computeclass for a workload in the format of workload=computeclass. Specify a single computeclass to set all workloads. (ex foo=example-class or example-class)
-  -i, --dev                       Enable interactive dev mode: build image, stream logs/status in the foreground and stop on exit
-  -e, --env strings               Environment variables to set on running containers
-  -f, --file string               Name of the build file (default "DIRECTORY/Acornfile")
-  -h, --help                      help for run
-      --interval string           If configured for auto-upgrade, this is the time interval at which to check for new releases (ex: 1h, 5m)
-  -l, --label strings             Add labels to the app and the resources it creates (format [type:][name:]key=value) (ex k=v, containers:k=v)
-      --link strings              Link external app as a service in the current app (format app-name:container-name)
-  -m, --memory strings            Set memory for a workload in the format of workload=memory. Only specify an amount to set all workloads. (ex foo=512Mi or 512Mi)
-  -n, --name string               Name of app to create
-      --notify-upgrade            If true and the app is configured for auto-upgrades, you will be notified in the CLI when an upgrade is available and must confirm it
-  -o, --output string             Output API request without creating app (json, yaml)
-      --profile strings           Profile to assign default values
-  -p, --publish strings           Publish port of application (format [public:]private) (ex 81:80)
-  -P, --publish-all               Publish all (true) or none (false) of the defined ports of application
-  -q, --quiet                     Do not print status
-      --region string             Region in which to deploy the app, immutable
-      --replace                   Replace the app with only defined values, resetting undefined fields to default values
-  -s, --secret strings            Bind an existing secret (format existing:sec-name) (ex: sec-name:app-secret)
-      --target-namespace string   The name of the namespace to be created and deleted for the application resources
-  -u, --update                    Update the app if it already exists
-  -v, --volume stringArray        Bind an existing volume (format existing:vol-name,field=value) (ex: pvc-name:app-data)
-      --wait                      Wait for app to become ready before command exiting (default true)
+      --auto-upgrade      Enabled automatic upgrades.
+  -f, --file string       Name of the build file (default "DIRECTORY/Acornfile")
+  -h, --help              help for run
+      --help-advanced     Show verbose help text
+  -n, --name string       Name of app to create
+      --notify-upgrade    If true and the app is configured for auto-upgrades, you will be notified in the CLI when an upgrade is available and must confirm it
```

Comment:
Don't think you need the "If true"

Could just be "If the app is configured...

Reply:
I think it makes sense to keep this as it is "in addition" to app being configured for auto-upgrades